### PR TITLE
:bug:Signed-Releases: really look for *.sign files

### PR DESCRIPTION
**Pull request by evverx**

### CODE REVIEW

**Review by laurentsimon — APPROVED** [review]
Thanks!

**Comment by evverx** [comment]
@laurentsimon the PR was out of date with the main branch so I rebased and pushed it once again disabling "auto-merge" along the way. I also updated the commit message to make it pass the "description" check. Could you take a look?

**Review by laurentsimon — APPROVED** [review]

**Comment by laurentsimon** [comment]
the integration tests are failing, we need to fix it upstream so the PR is blocked atm.

**Comment by evverx** [comment]
Just out of curiosity, is that "out-of-date" check I see every time the scorecard repository gets updated turned on via the "strict" status check mentioned at https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/about-protected-branches#require-status-checks-before-merging? It seems to be too annoying at least in projects receiving a lot of PRs so I'm not sure `scorecard` should penilize for not using it. It's totally understandable that maintainers wouldn't want to discourage drive-by contributors

**Comment by evverx** [comment]
I mean, it's probably OK to enforce it when all contributors work at the same company for example and have to follow this policy but it's just strange to enforce it on external contributors.

**Comment by evverx** [comment]
Just to clarify, I've opened 3 PRs and as far as I understand basically I'll have to rebase and push all of them one way or another once the integration test is fixed. Personally I think it's a waste of time given there are no any merge conflicts there

**Review by laurentsimon — APPROVED** [review]

File: checks/signed_releases.go
Comment (jessjon7792) [inline checks/signed_releases.go:31]: Var

**Pull request merged**
